src/elements/hero/src/stories.tsx

```diff
@@ -86,22 +86,26 @@ export const ExampleWithKnobs = () => {
   return (
     <div sx={{ margin: -10 }}>
       <Hero
-        contentWidth={contentWidth}
+        container={Container}
         fgImage={fgImage.img}
         fgImagePosition={fgImage.position}
         fgImageOnMobile={imageOnMobile}
         breadcrumbs={breadcrumbs}
       >
-        <Styled.h1>{headline}</Styled.h1>
-        <div sx={{ backgroundColor: 'white', padding: 20 }}>
-          <Styled.p sx={{ marginTop: 0 }}>
-            Lorem ipsum, or lipsum as it is sometimes known, is dummy text used
-            in laying out print, graphic or web designs.
-          </Styled.p>
-          <div sx={{ button: { width: 'auto' } }}>
-            <Button variant="primary">Optional CTA</Button>
-          </div>
-        </div>
+        <Row cols={20}>
```

Comment:
A 20 column grid? Shouldn't this be 12 like in the designs?

Reply:
yep, i was just converting the old version that was 45% width - will update to match the designs!